.cloud_build/flutter/Dockerfile

```diff
@@ -4,14 +4,15 @@
 FROM debian:bookworm-slim
 
 # Install Dependencies.
-RUN apt update -y
-RUN apt install -y \
-  git \
-  wget \
-  curl \
-  unzip \
-  lib32stdc++6 \
-  libglu1-mesa
+RUN apt-get update -y  \
+    && apt-get install -y \
+    git \
+    wget \
+    curl \
+    unzip \
+    lib32stdc++6 \
+    libglu1-mesa \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
nit: Should we run `apt-get clean` to delete package files also? I assume we're trying to reduce the container size?

Reply:
I'm just going off what gemini told me 😄 

Let me look...

Reply:
https://docs.docker.com/build/building/best-practices/#sort-multi-line-arguments

Reply:
Found more info. both is good!

Reply:
I guess I should just ask Gemini next time! Thanks for digging in.